localization/src/service.ts

```diff
@@ -206,6 +206,10 @@ export class LocalizationService<T> extends (LocalizedStringsService as Localize
         return Language.En;
     }
 
+    getFirstDayOfWeek = (): number => {
+        return this.localeInfo.dates.dayOfWeek || 1;
```

Comment:
Not sure about 1 as it's usually 0 AFAIR

Reply:
@savelichalex  0 is sunday

Reply:
I guess in USA (and some other countries) it's more common to start the week from Sunday. Agree with @savelichalex

Reply:
@sertony @savelichalex Are we focusing on America?
https://en.wikipedia.org/wiki/Week#:~:text=While%2C%20for%20example%2C%20the%20United,first%20day%20of%20the%20week.

Reply:
It's just what default in a browser, I would better stick to that

Reply:
@savelichalex which browser? By ISO first is monday

Reply:
chrome, btw you can use `Data` method as a fallback